ccpp/data/GFS_typedefs.meta

```diff
@@ -1860,6 +1860,78 @@
   dimensions = (horizontal_loop_extent)
   type = real
   kind = kind_phys
+[emdust]
+  standard_name = emission_of_dust_for_smoke
+  long_name = emission of dust for smoke
+  units = ug m-2 s-1
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  active = (do_smoke_coupling)
+[emseas]
+  standard_name = emission_of_seas_for_smoke
```

Comment:
I have no idea what `seas` is in this context. Although I know it makes the name longer, we should probably spell this out.